code-challenges.test.js

```diff
@@ -38,10 +72,36 @@ const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 // b) Create the function that makes the test pass.
 
+//create a function name oddLess
+//takes in one parameter 
+//use .filter method to remove anything that is not a number
+
+
+
+// const onlyOdds = fullArr1.filter(Number)
+
+
+const oddLess = (array) => {
+    let oddOnlys = (array)
+    return oddOnlys.filter()
+}
+
+// I am trying to pass in an array and return an array of only numbers
```

Comment:
Keep in mind that filter is a higher-order function and not a built in method, which means it will need an anonymous function where it knows what condition to filter.  This one will have a few conditions since you will need to remove all even numbers and check the typeof the current value.  You will then need to sort the numbers, highly recommend looking into the .sort() method.